src/bflib_math.h

```diff
@@ -75,6 +75,11 @@ TbBool LbNumberSignsSame(long num_a, long num_b);
 char LbCompareMultiplications(long mul1a, long mul1b, long mul2a, long mul2b);
 long LbDiagonalLength(long a, long b);
 
+DLLIMPORT extern int _DK_lbCosTable[2048];
+#define lbCosTable _DK_lbCosTable
+DLLIMPORT extern int _DK_lbSinTable[2048];
+#define lbSinTable _DK_lbSinTable
+
```

Comment:
as explained in other comment just use the function, no changes needed in blib_math